src/main/java/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCMSource.java

```diff
@@ -301,6 +308,19 @@ protected SCMRevision retrieve(SCMHead head, TaskListener listener)
                     ((BitbucketBranchSCMHead) head).getLatestCommit());
```

Comment:
Thanks for fixing the tag SCM head. The two above are still incorrect- out of the scope for this PR but can we add a comment like
```code
                    // JENKINS-73267 fetch commit from source rather than cached value
```